src/sdk/SpeechTranslationConfig.ts

```diff
@@ -68,6 +71,8 @@ export abstract class SpeechTranslationConfig extends SpeechConfig {
      *       "language=de-DE", and also set by the speechRecognitionLanguage property, the language
      *       setting in uri takes precedence, and the effective language is "de-DE".
      * Only the properties that are not specified in the endpoint URL can be set by other APIs.
+     * Note: To use authorization token with fromEndpoint, pass an empty string to the subscriptionKey in the fromEndpoint method,
+     * and then call setAuthorizationToken() on the created SpeechConfig instance to use the authorization token.
```

Comment:
>setAuthorizationToken() [](start = 21, length = 23)

cf. elsewhere. if there's a real property and no setter, should be changed

Reply:
fixed